R/interventions.R

```diff
@@ -238,12 +238,13 @@ add_smc <- function(p, interventions){
 #' @inheritParams add_interventions
 #'
 #' @return modified parameter list
-add_rtss <- function(p, interventions){
+add_pev <- function(p, interventions){
   month <- 365 / 12
   timesteps <- 1 + (interventions$year - p$baseline_year) * 365
 
-  p <- malariasimulation::set_rtss_epi(
+  p <- malariasimulation::set_pev_epi(
     parameters = p,
+    profile = rtss_profile,
     timesteps = timesteps,
     coverages = interventions$rtss_cov,
     age = round(6 * month),
```

Comment:
There are some pieces missing here, if we look at https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html#simulation-2 then we need to:

- [ ]  additionally specify the `booster_profile`
- [ ] rename the `boosters` argument to `booster_timestep`
- [ ] `seasonal_boosters` is by default set to `FALSE` so we can remove that